tests/platform_tests/mellanox/test_thermal_control.py

```diff
@@ -4,17 +4,18 @@
 import random
 import time
 from tests.common.mellanox_data import get_platform_data
-from tests.common.plugins.loganalyzer.loganalyzer import LogAnalyzer
 from tests.common.utilities import wait_until
 from tests.platform_tests.thermal_control_test_helper import *
 from mellanox_thermal_control_test_helper import MockerHelper, AbnormalFanMocker
+from tabulate import tabulate
+import re
 
 pytestmark = [
     pytest.mark.asic('mellanox'),
     pytest.mark.topology('any')
 ]
 
-THERMAL_CONTROL_TEST_WAIT_TIME = 65
+THERMAL_CONTROL_TEST_WAIT_TIME = 75
```

Comment:
why more time is needed?

Reply:
Thermal loop usually takes 60 seconds, but we found some slow sysfs read in a few testbeds, in that case, wait 65 seconds is not enough. We talked to low level team, they claimed that the slow sysfs read is not an issue, so we have to enhance the wait time here.